### PR TITLE
Update redirected URLs (part 33)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Extending #14055

## Summary

Updated redirected URLs.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/glossary/node.js/index.md [hunk 1]
```diff
@@ -18,5 +18,5 @@ Node.js is a cross-platform {{Glossary("JavaScript")}} runtime environment that
 - {{Interwiki("Wikipedia", "Node.js")}} on Wikipedia
 - [Node.js website](https://nodejs.org/)
 - [API reference documentation](https://nodejs.org/api/)
-- [Tutorials](https://nodejs.org/documentation/tutorials/)
+- [Tutorials](https://nodejs.org/en/docs/)
```
Comment (OnkarRuikar) [inline files/en-us/glossary/node.js/index.md:21]:
```suggestion
- [Guides](https://nodejs.org/en/docs/guides/)
```

File: files/en-us/learn/html/multimedia_and_embedding/video_and_audio_content/index.md [hunk 2]
```diff
@@ -295,7 +295,7 @@ And that's a wrap — we hope you had fun playing with video and audio in web pa
 - [Adding captions and subtitles to HTML5 video](/en-US/docs/Web/Guide/Audio_and_video_delivery/Adding_captions_and_subtitles_to_HTML5_video)
 - [Audio and Video delivery](/en-US/docs/Web/Guide/Audio_and_video_delivery): A LOT of detail about putting audio and video onto web pages using HTML and JavaScript.
 - [Audio and Video manipulation](/en-US/docs/Web/Guide/Audio_and_video_manipulation): A LOT of detail about manipulating audio and video using JavaScript (for example adding filters.)
-- Automated options to [translate multimedia](https://www.inwhatlanguage.com/blog/translate-video-audio/).
+- Automated options to [translate multimedia](https://www.inwhatlanguage.com/translate-video-audiotranslate-video-audio-2/).
```
Comment (OnkarRuikar) [inline files/en-us/learn/html/multimedia_and_embedding/video_and_audio_content/index.md:298]: It's an empty post now.
```suggestion
```

File: files/en-us/learn/performance/multimedia/index.md [hunk 3]
```diff
@@ -109,7 +109,7 @@ In image delivery the "one size fits all" approach will not yield the best resul
 
 Two interesting effects to keep in mind regarding high dpi screens is that:
 
-- with high DPI screen, [humans will spot compression artifacts a lot later](https://www.netvlies.nl/tips-updates/algemeen/design-interactie/retina-revolution/), meaning that for images meant for these screens you can crank up compression beyond usual.
+- with high DPI screen, [humans will spot compression artifacts a lot later](https://www.netvlies.nl/blogs/retina-revolution), meaning that for images meant for these screens you can crank up compression beyond usual.
```
Comment (OnkarRuikar) [inline files/en-us/learn/performance/multimedia/index.md:112]: barred behind access/login
```suggestion
- with a high DPI screen, humans will spot compression artifacts a lot later, meaning that for images meant for these screens you can crank up compression beyond usual.
```

File: files/en-us/learn/server-side/express_nodejs/mongoose/index.md [hunk 4]
```diff
@@ -124,7 +124,7 @@ Mongoose is installed in your project (**package.json**) like any other dependen
 npm install mongoose
 ```
 
-Installing _Mongoose_ adds all its dependencies, including the MongoDB database driver, but it does not install MongoDB itself. If you want to install a MongoDB server then you can [download installers from here](https://www.mongodb.com/download-center) for various operating systems and install it locally. You can also use cloud-based MongoDB instances.
+Installing _Mongoose_ adds all its dependencies, including the MongoDB database driver, but it does not install MongoDB itself. If you want to install a MongoDB server then you can [download installers from here](https://www.mongodb.com/try) for various operating systems and install it locally. You can also use cloud-based MongoDB instances.
```
Comment (OnkarRuikar) [inline files/en-us/learn/server-side/express_nodejs/mongoose/index.md:127]:
```suggestion
Installing _Mongoose_ adds all its dependencies, including the MongoDB database driver, but it does not install MongoDB itself. If you want to install a MongoDB server then you can [download installers from here](https://www.mongodb.com/try/download/community) for various operating systems and install it locally. You can also use cloud-based MongoDB instances.
```

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md [hunk 5]
```diff
@@ -275,7 +275,7 @@ You should seriously consider the support matrix and DSLs of a framework when ma
 Citations for this table:
 
 - [React browser support: official docs](https://reactjs.org/docs/react-dom.html#browser-support)
-- [Ember browser support: Ember 3.0 release announcement](https://blog.emberjs.com/2018/02/14/ember-3-0-released.html)
+- [Ember browser support: Ember 3.0 release announcement](https://blog.emberjs.com/ember-3-0-released/)
```
Comment (OnkarRuikar) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:278]:
```suggestion
- [Ember browser support: Ember 3.0 Released](https://blog.emberjs.com/ember-3-0-released/)
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/javascript/index.md [hunk 6]
```diff
@@ -392,7 +392,7 @@ Note that polyfills.js is basically the two polyfills we are using put together
...
 
-> **Note:** There are some 3rd party options to consider, for example [Polyfill.io](https://polyfill.io/v2/docs/) — this is a meta-polyfill library that will look at each browser's capabilities and apply polyfills as needed, depending on what APIs and JS features you are using in your code.
+> **Note:** There are some 3rd party options to consider, for example [Polyfill.io](https://polyfill.io/v3/) — this is a meta-polyfill library that will look at each browser's capabilities and apply polyfills as needed, depending on what APIs and JS features you are using in your code.
```
Comment (OnkarRuikar) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/javascript/index.md:395]:
```suggestion
> **Note:** There are some 3rd party options to consider, for example [Polyfill.io](https://polyfill.io/v3/api/) — this is a meta-polyfill library that will look at each browser's capabilities and apply polyfills as needed, depending on what APIs and JS features you are using in your code.
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md [hunk 7]
```diff
@@ -603,9 +603,9 @@ Let's write an example:
...
 
-> **Note:** If you don't want to write out the capabilities objects for your tests by hand, you can generate them using the generators embedded in the docs. See [Run tests on mobile browsers](https://www.browserstack.com/automate/node#run-tests-on-mobile) and [Run tests on desktop browsers](https://www.browserstack.com/automate/node#setting-os-and-browser).
+> **Note:** If you don't want to write out the capabilities objects for your tests by hand, you can generate them using the generators embedded in the docs. See [Run tests on mobile browsers](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#run-tests-on-mobile) and [Run tests on desktop browsers](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#setting-os-and-browser).
```
Comment (OnkarRuikar) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md:608]:
```suggestion
> **Note:** If you don't want to write out the capabilities objects for your tests by hand, you can generate them using the generators embedded in the docs. See [Run your first test](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#run-your-first-test).
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**